packages/visx-axis/test/Axis.test.tsx

```diff
@@ -66,7 +66,7 @@ describe('<Axis />', () => {
     expect(wrapper.find(`.${tickClassName}`).length).toBeGreaterThan(0);
   });
 
-  test('it should pass the output of tickLabelProps to tick labels', () => {
```

Comment:
have been updating `test('it ...')` to `it('...')` as I've encountered them when updating tests